tsfc/tensor.py

```diff
@@ -83,7 +83,7 @@ def classify(quadrature_indices, expression):
     return result
 
 
-def flatten(var_reps):
+def flatten(var_reps, index_cache):
     for variable, reps in var_reps:
         expressions = reps
         for expression in expressions:
```

Comment:
No unconcat handling in tensor because it won't work, or because it's not implemented yet?